source/about/mattermost-v10-changelog.md

```diff
@@ -3,6 +3,94 @@
 ```{Important}
 ```{include} common-esr-support-upgrade.md
 ```
+(release-v10.9-feature-release)=
+## Release v10.9 - [Feature Release](https://docs.mattermost.com/about/release-policy.html#release-types)
+
+**Release day: 2025-06-16**
+
+### Compatibility
+ - Updated the minimum supported versions of Edge and Chrome to 134+, and Firefox to 128+.
+
+### Important Upgrade Notes
+ - A new index to the ``CategoryId`` column in ``SidebarChannels`` table was added to improve query performance. No database downtime is expected for this upgrade. See the [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html) for more details.
+
+```{Important}
+If you upgrade from a release earlier than v10.8, please read the other [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html).
+```
+
+### Improvements
+
+#### User Interface (UI)
+ - Consolidated all channel editing functionalities into a single, accessible modal located in the channel header menu. Users can now update channel names, URL slugs, convert between public and private, modify/add a purpose and header (with a live markdown preview), manage channel banners, and archive the channel—all in one place. Updates include safeguards for unsaved edits, improved URL-slug editing, and enhanced keyboard and navigation accessibility.
+ - Pre-packaged MS Teams plugin [v2.2.1](https://github.com/mattermost/mattermost-plugin-msteams/releases/tag/v2.2.1).
+ - Pre-packaged Playbooks plugin [v2.2.0](https://github.com/mattermost/mattermost-plugin-playbooks/releases/tag/v2.2.0).
+ - Pre-packaged Calls plugin [v1.8.0](https://github.com/mattermost/mattermost-plugin-calls/releases/tag/v1.8.0).
+ - Pre-packaged Jira plugin version [v4.3.0](https://github.com/mattermost/mattermost-plugin-jira/releases/tag/v4.3.0).
+ - Introduced a configurable channel banner feature for channel admins, visible across desktop, web, and mobile platforms. This feature requires an Enterprise Advanced license.
```

Comment:
Should we separately announce the Enterprise Advanced License? When we do, can we mention that it is Postgres-only?

Reply:
@cwarnermm Do we have anywhere in the documentation that we can outline some requirements for the new sku? We need to highlight: 

- From which MM Version is supported (v10.9)
- Highlight that enterprise plugins would need to be updated to support the. new license as well (most prepackaged in that version) 
- Database requirement.

Reply:
Added a line in the "Administration" section for the new SKU.